salt/apache/files/sites/docs.conf.include

```diff
@@ -76,8 +76,6 @@ SetEnv BANNER /includes/banner_live.html
     SetEnv BANNER /includes/banner_staging_profiles_ppp.html
 </Location>
 
-# Needed to proxy to SSL servers.
-SSLProxyEngine on
 # ProxyPreserveHost was on but we need it to be Off, as it makes adding SSL much easier.
 ProxyPreserveHost Off
 # With a keepalive, we had problems with headers being interpreted as the start of the response.
```

Comment:
I guess we don't need these lines either, anymore.

Reply:
Agreed, removed in new commit.